bin/GameIndex.dbf

```diff
@@ -36354,7 +36400,8 @@ Compat = 5
 Serial = SLUS-20546
 Name   = Evolution Snowboarding
 Region = NTSC-U
-Compat = 3
+// Need the EE set to interpreter + cache enable
+OPHFLagHack = 1 // avoid hanging at the "press start" screen
```

Comment:
You removed the compatibility status for this one. Make sure to add it back as `Compat=5`

Reply:
Then you need to get the status on the website updated first. It's still marked as in-game even though you added a playable report on the forum.

> Need the EE set to interpreter + cache enable

Needs EmotionEngine set to "Interpreter" and "Enable EE Cache" enabled.

For consistancy, begin the comment OPHFLagHack with a capital letter too (Avoid).